src/app/containers/CpsRecommendations/RecommendationsPromoList/index.jsx

```diff
@@ -41,7 +58,11 @@ const RecommendationsPromoListItem = forwardRef(
   },
 );
 
-const RecommendationsPromoList = ({ promoItems }) => {
+const RecommendationsPromoList = ({
+  promoItems,
+  showForVariation,
+  splitRecsViewEventTracker,
```

Comment:
Can the Optimizely context for the event tracking be declared inside here rather than passing it down from StoryPage? Just trying to follow the flow of it.

Reply:
I called it in StoryPage because we only want to send one view event if either block is viewed . If I put the view track inside cps, it would sent two events when both blocks have been viewed (this is because we call cps twice, for each part). I tried to avoid called it in StoryPage but couldn't come up with a better solution yet.

Reply:
Ah right I see, no worries 👍🏼